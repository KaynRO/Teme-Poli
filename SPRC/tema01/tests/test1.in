login kayn
load
add 1 1 1
add 2 2 1.2 2.2
add 1 2 4.5 5.5
read 1
read 2
read 3
update 1 3 3.3 4.4 5.5
update 3 2 9.9 8.8
read 3
store
logout
