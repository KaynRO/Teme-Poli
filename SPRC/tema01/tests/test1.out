--------------------------------
Session key: 1605611524
--------------------------------
Load successfull
--------------------------------
Data Add successfull
--------------------------------
Data Add successfull
--------------------------------
Data ID does not/already exists
--------------------------------
Data ID: 1
No Values: 1
Values: 1.000000
--------------------------------
Data ID: 2
No Values: 2
Values: 1.200000 2.200000
--------------------------------
Data ID does not exists
--------------------------------
Update successfull
--------------------------------
Update successfull
--------------------------------
Data ID: 3
No Values: 2
Values: 9.900000 8.800000
--------------------------------
Store successfull
--------------------------------
Logout successfully
